src/config/API_URL.ts

```diff
@@ -1,3 +1,4 @@
 export const API_URL = 'https://itunes.apple.com/search';
 export const DETAILS_URL = 'https://itunes.apple.com/lookup';
 export const NUMBER_OF_RESULTS = '50';
+export const RANDOM_ARTIST_API_URL = 'https://itunes.apple.com/us/rss/topsongs/limit=10/json';
```

Comment:
There aren't much of free resources left on the api, so rss is a good choice. The new one, [Apple Music API](https://developer.apple.com/documentation/applemusicapi), is a paid service